include/amici/solver.h

```diff
@@ -48,6 +48,8 @@ class Solver {
         sensi = other.sensi;
         atol = other.atol;
         rtol = other.rtol;
+        atol_sensi = other.atol_sensi;
+        rtol_sensi = other.rtol_sensi;
```

Comment:
Okay, as hotfix, but better handling of the solver memory would avoid dealing with the long list of members in copy, copyassign, move, equality, ..... Which will get out of sync soon again ;). I guess the same members are also still missing in serialization functions.